3DAmsterdam/Assets/Netherlands3D/Scripts/ObjectInteraction/Transformable.cs

```diff
@@ -221,7 +243,7 @@ private void FollowMousePointer()
 
 			//offset.x = -bounds.min.x % 100;
 			//offset.z = -bounds.min.z % 100;
-			if (placeOnGrid)
+			if (snapToGrid)
 			{
 				newPosition.x -= ((newPosition.x + bounds.min.x) % 100);
```

Comment:
misschien handig hier ook de gridsize van de visualgrid te gebruiken in plaats van een hardcoded gridmaat?

Reply:
ahh goeie. zal m' er bij zetten